atlasdb-config/src/main/java/com/palantir/atlasdb/factory/timelock/TimestampCorroboratingTimelockService.java

```diff
@@ -67,6 +68,13 @@ public StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransacti
                 r -> r.startTimestampAndPartition().timestamp());
     }
 
+    @Override
+    public StartIdentifiedAtlasDbTransactionResponseBatch startIdentifiedAtlasDbTransactionsBatch(int count) {
```

Comment:
Again, would it make sense to only keep the batched method? count=1 should be simple to handle.